surefire-provider/src/test/java/org/arquillian/smart/testing/surefire/provider/LoaderVersionExtractorSurefireJUnit5Test.java

```diff
@@ -1,50 +0,0 @@
-package org.arquillian.smart.testing.surefire.provider;
-
-import java.io.File;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.URLClassLoader;
-import java.util.Arrays;
-import java.util.Collection;
-import org.jboss.shrinkwrap.resolver.api.maven.Maven;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-@RunWith(Parameterized.class)
-public class LoaderVersionExtractorSurefireJUnit5Test {
```

Comment:
Those two tests were removed in favor of `JUnit5ProviderInfoTest`?

Reply:
nope, there have been removed methods that were getting the version so the tests are not needed https://github.com/arquillian/smart-testing/pull/246/files#diff-f265edf8b316db4ccfc2829915f4af85L46